configs/5.5/ffigen.yaml

```diff
@@ -3,135 +3,139 @@ preamble: |
-  - "-I./rootstraps/5.5/usr/include/sensor/"
-  - "-I./rootstraps/5.5/usr/include/shortcut/"
-  - "-I./rootstraps/5.5/usr/include/storage/"
-  - "-I./rootstraps/5.5/usr/include/system/"
-  - "-I./rootstraps/5.5/usr/include/web/"
-  - "-I./rootstraps/5.5/usr/include/wifi-direct/"
-  - "-I./rootstraps/5.5/usr/include/yaca/"
+  - '-m32'
+  - '-Wno-incomplete-setjmp-declaration'
+  - '-I./rootstraps/5.5/usr/include/'
+  - '-I./rootstraps/5.5/usr/include/appcore-agent/'
+  - '-I./rootstraps/5.5/usr/include/appfw/'
+  - '-I./rootstraps/5.5/usr/include/asp/'
+  - '-I./rootstraps/5.5/usr/include/badge/'
+  - '-I./rootstraps/5.5/usr/include/cbhm/'
+  - '-I./rootstraps/5.5/usr/include/ckm/'
+  - '-I./rootstraps/5.5/usr/include/content/'
+  - '-I./rootstraps/5.5/usr/include/context-service/'
+  - '-I./rootstraps/5.5/usr/include/csr/'
+  - '-I./rootstraps/5.5/usr/include/device-certificate-manager/'
+  - '-I./rootstraps/5.5/usr/include/dlog/'
+  - '-I./rootstraps/5.5/usr/include/eom/'
+  - '-I./rootstraps/5.5/usr/include/feedback/'
+  - '-I./rootstraps/5.5/usr/include/geofence/'
+  - '-I./rootstraps/5.5/usr/include/iotcon/'
+  - '-I./rootstraps/5.5/usr/include/location/'
+  - '-I./rootstraps/5.5/usr/include/media/'
+  - '-I./rootstraps/5.5/usr/include/media-content/'
+  - '-I./rootstraps/5.5/usr/include/messaging/'
+  - '-I./rootstraps/5.5/usr/include/metadata-editor/'
+  - '-I./rootstraps/5.5/usr/include/network/'
+  - '-I./rootstraps/5.5/usr/include/notification/'
+  - '-I./rootstraps/5.5/usr/include/notification-ex/api/'
+  - '-I./rootstraps/5.5/usr/include/nsd/'
+  - '-I./rootstraps/5.5/usr/include/privacy-privilege-manager/'
+  - '-I./rootstraps/5.5/usr/include/rpc-port/'
+  - '-I./rootstraps/5.5/usr/include/sensor/'
+  - '-I./rootstraps/5.5/usr/include/shortcut/'
+  - '-I./rootstraps/5.5/usr/include/storage/'
+  - '-I./rootstraps/5.5/usr/include/system/'
+  - '-I./rootstraps/5.5/usr/include/web/'
+  - '-I./rootstraps/5.5/usr/include/wifi-direct/'
+  - '-I./rootstraps/5.5/usr/include/yaca/'
 
   # include efl directories
-  - "-I./rootstraps/5.5/usr/include/evas-1/"
-  - "-I./rootstraps/5.5/usr/include/efl-1/"
-  - "-I./rootstraps/5.5/usr/include/eo-1/"
-  - "-I./rootstraps/5.5/usr/include/eina-1/"
-  - "-I./rootstraps/5.5/usr/include/eina-1/eina/"
-  - "-I./rootstraps/5.5/usr/include/emile-1/"
-  - "-I./rootstraps/5.5/usr/include/ecore-imf-1/"
+  - '-I./rootstraps/5.5/usr/include/evas-1/'
+  - '-I./rootstraps/5.5/usr/include/efl-1/'
+  - '-I./rootstraps/5.5/usr/include/eo-1/'
+  - '-I./rootstraps/5.5/usr/include/eina-1/'
+  - '-I./rootstraps/5.5/usr/include/eina-1/eina/'
+  - '-I./rootstraps/5.5/usr/include/emile-1/'
+  - '-I./rootstraps/5.5/usr/include/ecore-imf-1/'
 
   # include glib directories
-  - "-I./rootstraps/5.5/usr/include/glib-2.0/"
-  - "-I./rootstraps/5.5/usr/lib/glib-2.0/include/"
+  - '-I./rootstraps/5.5/usr/include/glib-2.0/'
+  - '-I./rootstraps/5.5/usr/lib/glib-2.0/include/'
 
-dart-bool: true
+structs:
+  dependency-only: opaque
```

Comment:
Do you have an example of what this exactly does?

Reply:
This replace unused structure to ffi.Opaque like `Eina_List`. We can save few lines...